task_sdk/src/airflow/sdk/definitions/connection.py

```diff
@@ -57,6 +58,18 @@ def get_uri(self): ...
 
     def get_hook(self): ...
 
+    @classmethod
+    def get(cls, conn_id: str, default_conn: Any = None) -> Any:
```

Comment:
I didnt think we have a default on get connection do we?

Reply:
I didnt think either, but seems like it was inspired from whats present in the ConnectionAccessor.
https://github.com/apache/airflow/blob/main/airflow/utils/context.py#L130-L146

Do you think there's value to keep it when used with basehook? I dont think there's any value.

Reply:
We decided to continue with Connection.get instead as Connection is a new interface & we can use this opportunity to make things simpler. Connection.get is simpler than Connection.get_connection_from_secrets